test/platform/github/index.spec.ts

```diff
@@ -1,5 +1,5 @@
 import fs from 'fs-extra';
-import { GotApi, GotResponse, RepoConfig } from '../../../lib/platform/common';
+import { GotApi, GotResponse, RepoConfig } from '../../../lib/platform';
```

Comment:
Why this change?

Reply:
IDE suggested that improvement, as there is only one file under the platform folder, that is imported automatically.